Sources/OpenCombine/Published.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Published.swift
+//  OpenCombine
+//
+//  Created by Евгений Богомолов on 01/09/2019.
+//
+
+import Foundation
+
+@propertyWrapper public struct Published<Value> {
+
+    /// Initialize the storage of the Published property as well as the corresponding `Publisher`.
+    public init(initialValue: Value) {
+        self.value = initialValue
+    }
+
+    public struct Publisher : OpenCombine.Publisher {
+
+        /// The kind of values published by this publisher.
+        public typealias Output = Value
+
+        /// The kind of errors this publisher might publish.
+        ///
+        /// Use `Never` if this `Publisher` does not publish errors.
+        public typealias Failure = Never
+
+        /// This function is called to attach the specified `Subscriber` to this `Publisher` by `subscribe(_:)`
+        ///
+        /// - SeeAlso: `subscribe(_:)`
+        /// - Parameters:
+        ///     - subscriber: The subscriber to attach to this `Publisher`.
+        ///                   once attached it can begin to receive values.
+        public func receive<S>(subscriber: S) where Value == S.Input, S : Subscriber, S.Failure == Published<Value>.Publisher.Failure {
+            
+            _passthrowObject.receive(subscriber: subscriber)
```

Comment:
It is [discouraged](https://developer.apple.com/documentation/combine/publisher/3204756-subscribe) to call this method, can you please change it to `_passthroughSubject.subscribe(subscriber)`?